### ISSUE TITLE
about fix previous head at the second stage 

### ISSUE THREAD

**Issue by fafafafafafafa** [xialeiliu/Long-Tailed-CIL#3]
您好，在您的论文中，公式(3)下面说："We found it to be essential to fix previous head $h_{ϕ1:t−1}$ at the second stage
when learning together with LWS layer"
然而，我在对照eeil_2stage.py 中发现，您的代码是这样实现的: `output.append(self.model.heads[idx](feat.detach()))`
关于这段代码，我不是很理解，您能给我解释一下吗？

**Comment by Ethanhuhuhu** [comment]
optimizer当中不更新先前任务的head，这里detach只是保险
https://github.com/xialeiliu/Long-Tailed-CIL/blob/7db63177de101071b5354f689ec71e0dbe0416f2/src/approach/eeil_2stage.py#L55-L58
